plugins/builder/osbuild.py

```diff
@@ -80,11 +80,22 @@ def as_dict(self, arch: str = ""):
 
 
 class Repository:
-    def __init__(self, baseurl: str, gpgkey: str = None):
+    def __init__(self, baseurl: str):
         self.baseurl = baseurl
-        self.gpgkey = gpgkey
+        self.gpgkey = None
+        self.package_sets: List[str] = None
         self.rhsm = False
 
+    @classmethod
+    def from_data(cls, data: Union[str, Dict]) -> "Repository":
+        print(data)
```

Comment:
Leftover print or do you want this here?

Reply:
Yes.

Reply:
Fixed ;)